recipe/bld.bat

```diff
@@ -5,4 +5,4 @@ echo udunits2_path = %SCRIPTS%\udunits2.dll >> %SITECFG%
 rmdir lib\iris\tests\results /s /q
 del lib\iris\tests\*.npz
 
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Actually that ended up being a wild goose chase in matplolitb. The real issue there was something :unamused: